@hubblecommerce/hubble/core/components/swComponents/slots/cross-selling-slot.vue

```diff
@@ -11,5 +11,11 @@ export default {
     name: 'CrossSellingSlot',
 
     mixins: [slotMixins],
+
+    computed: {
+        crossSellings() {
+            return this.content && this.content.data && this.content.data.crossSellings;
```

Comment:
Returns false in worst case, property is defined as array in cross sellings component

Reply:
Fixed in [4f6a2c0](https://github.com/hubblecommerce/hubble-frontend-pwa/commit/4f6a2c0daa4dab222881bbab35de3721d6564ed9)